src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -282,6 +282,15 @@ const generateTaskRole = (taskRole, labels, config) => {
     gangAllocation = 'false';
     retryPolicy.fancyRetryPolicy = true;
   }
+  // calculate pod priority
+  // reference: https://github.com/microsoft/pai/issues/3704
+  let jobPriority = 127;
```

Comment:
>127 [](start = 20, length = 3)

If no hived enabled, let's treat the same as user not specify, i.e. default to jobPriority = 126 - 0

Reply:
updated

Reply:
I suggest that you can first preprocess the hivedPodSpec.priority
1. If hived not enable, jobPriority = 0
2. else jobPriority = round (hivedPodSpec.priority)
Then reversejobPriority = 126 - jobPriority
Is it more clear?